src/classes/Lists.ts

```diff
@@ -19,4 +26,154 @@ export default class Lists extends OneBlinkAPI {
     options = options || {}
     super(options.accessKey, options.secretKey)
   }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const searchParams = {
+   *   limit: 1,
+   *   offset: 0,
+   * }
+   * const { formElementLists, meta } = await lists.searchLists(
+   *   searchParams,
+   * )
+   * ```
+   *
+   * @param searchParams Search options
+   */
+  async searchLists(searchParams: ListSearchOptions): Promise<
+    Omit<ListSearchResult, 'formElementDynamicOptionSets'> & {
+      formElementLists: FormTypes.FormElementOptionSet[]
+    }
```

Comment:
Can we swap this so that this returns `ListSearchResult` and 

```
const result = await super.searchRequest<ListSearchResult>(
```

becomes

```
const result = await super.searchRequest<Omit<ListSearchResult, 'formElementLists'> & {
      formElementDynamicOptionSets: FormTypes.FormElementOptionSet[]
    }>(
```

Reply:
Yes